apis/object/v1alpha1/conversion.go

```diff
@@ -0,0 +1,59 @@
+package v1alpha1
+
+import (
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"sigs.k8s.io/controller-runtime/pkg/conversion"
+
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+
+	"github.com/crossplane-contrib/provider-kubernetes/apis/object/v1beta1"
+)
+
+// ConvertTo converts this Object to the Hub version (v1beta1).
+func (src *Object) ConvertTo(dstRaw conversion.Hub) error { // nolint:golint // We want to use different names for receiver parameter to be more clear.
+	dst := dstRaw.(*v1beta1.Object)
+
+	switch src.Spec.ManagementPolicy {
+	case Default:
+		dst.Spec.ManagementPolicies = xpv1.ManagementPolicies{xpv1.ManagementActionAll}
+	case ObserveCreateUpdate:
+		dst.Spec.ManagementPolicies = xpv1.ManagementPolicies{xpv1.ManagementActionObserve, xpv1.ManagementActionCreate, xpv1.ManagementActionUpdate}
```

Comment:
Should we include LateInitialize here? What was the default behaviour of ObserveCreateUpdate, did it do some LateInitialization?